noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/AsterixHttpRequest.scala

```diff
@@ -0,0 +1,39 @@
+package edu.uci.ics.cloudberry.noah.feed
+
+import java.net.URLEncoder
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import play.api.libs.ws.ahc.AhcWSClient
+import play.api.libs.ws.ning.NingWSClient
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object AsterixHttpRequest {
+
+  def insertDB(server: String, dataverse:String, dataset:String, record:String){
+    var adm = TagTweet.tagOneTweet(record, false)
+    adm = URLEncoder.encode(adm, "UTF-8")
+    val url = server + "/update?statements=use dataverse "+dataverse+"; insert into dataset "+dataset+"("+adm+");"
```

Comment:
one trick about scala, you can write formatted string as

`s"$server/update?statements=use dataverse $dataverse + ; insert into dataset $dataset ( $adm);"`